api/src/services/listing-csv-export.service.ts

```diff
@@ -851,6 +851,7 @@ export class ListingCsvExporterService implements CsvExporterServiceInterface {
       user &&
       (user.userRoles?.isAdmin ||
         user.userRoles?.isJurisdictionalAdmin ||
+        user.userRoles?.isLimitedJurisdictionalAdmin ||
```

Comment:
Is the new role meant to be able to export the CSV? We have a new test `should error as forbidden for csv endpoint`

Reply:
The new role should be able to export the listing csv